modules/ethereum/src/lib.rs

```diff
@@ -251,9 +308,24 @@ pub trait Trait: frame_system::Trait {
 
 decl_module! {
 	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
-		/// Import Aura chain headers. Ignores non-fatal errors (like when known
-		/// header is provided), rewards for successful headers import and penalizes
-		/// for fatal errors.
+		/// Import single Aura header. Requires transaction to be **UNSIGNED**.
+		pub fn unsigned_import_header(origin, header: Header, receipts: Option<Vec<Receipt>>) {
```

Comment:
I was reading through the code again and managed to confuse myself for a moment thinking that this function called `import_header`**s** instead of `import_header`. I think it'd be less confusing to rename the functions to:

`unsigned_import_header` => `import_unsigned_header`
`import_headers` => `import_signed_headers`